app/dao/users_dao.py

```diff
@@ -82,6 +82,14 @@ def delete_user_verify_codes(user):
     db.session.commit()
 
 
+def count_user_verify_codes(user):
+    query = db.session.query(
+        func.count().label('count')
+    ).filter(VerifyCode.user == user,
+             VerifyCode.expiry_datetime <= datetime.utcnow()).one()
```

Comment:
also, while you're looking at this fn, this can be done by just saying `return VerifyCode.query.filter(....).count()`, rather than needing .one and the label and all that gubbins